### ISSUE TITLE
Should 'find' fetch results from the output?

### ISSUE THREAD

**Issue by sanketd11** [att/rcloud#1297]
Currently only the results from the code are highlighted.
User may want to find from the output as well once the code is executed.

**Comment by gordonwoodhull** [comment]
This would be great but I was focusing mostly on search and replace, and obviously results can't be modified.  Will have to put off for 1.4.

**Comment by anatoliyg** [comment]
just to confirm, we'd want to replace the results of any executions only on the front end? Should the replacements persist if you reload the page or re-run the cells?

**Comment by gordonwoodhull** [comment]
I don't think we should allow replacements in the results, just searching. Otherwise one would be literally falsifying results. So if you're in replace mode, the replace button/command should not work whenever in a result area, the only thing to do on those finds is go to next.

**Comment by anatoliyg** [comment]
Makes sense.thanks

**Comment by s-u** [comment]
It would be nice if find was global (not just code and/or results), e.g. so it would work in the notebook tree as well (so you can search for a notebook name, user name etc.) It may require some separation of the search functions or some notion of focus.

The current behavior is not unreasonable and is very helpful as a search within the notebook, but from a browser-user perspective one would expect to be able to use `<Cmd><F>` to find any content on the page.

**Comment by anatoliyg** [comment]
I think that would be a good feature too. And should be doable on the FE
alone, since things like username and notebook name are present in the
tree's html, so we won't have to search any models for matches, just the
dom, then highlight tree's nodes and sub-nodes and scroll them into view
when a match is found. And maybe we can add some notion of focus to the
current cmd+F prompt.

**Comment by gordonwoodhull** [comment]
It's a good idea but it really merits a separate issue because it would require work unrelated to what this issue is about. I don't think the browser's built-in search would help, so it's a matter of implementing incremental search on top of the jquery.tree.
